src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+const PersonPartner = ({ person }) => (
+  <p className="Person__partner">
+    {person.sex === 'm' && person.isMarried === true ? `${person.partnerName} is my wife` : ''}
+
+    {person.sex === 'f' && person.isMarried === true ? `${person.partnerName} is my husband` : ''}
```

Comment:
These two lines look like duplications.
You can create a separate variable and calculate person `partner `based on `person.sex` field